app/app.go

```diff
@@ -65,7 +66,7 @@ var (
 	// DefaultNodeHome default home directories for the application daemon
 	DefaultNodeHome string
 
-	Upgrades = []upgrades.Upgrade{v3.Upgrade, v3_rc3.Upgrade}
+	Upgrades = []upgrades.Upgrade{v3.Upgrade, v3_rc3.Upgrade, v3_rc4.Upgrade}
```

Comment:
Or we replace rc4 to rc3?

Reply:
Now our testnet is running on rc3 version